docs/_ext/schemagen.py

```diff
@@ -11,7 +11,7 @@
 class SchemaGen(SphinxDirective):
 
     def run(self):
-        self.env.note_dependency('../siliconcompiler/schema.py')
+        self.env.note_dependency('../siliconcompiler/schema_cfg.py')
```

Comment:
Isn't this missing something, isn't it 'siliconcompiler/schema/schema_cfg.py'?

Reply:
yes, you are correct